lib/cloud_controller/dependency_locator.rb

```diff
@@ -201,6 +201,10 @@ def paginated_collection_renderer
       create_paginated_collection_renderer
     end
 
+    def created_object_renderer
```

Comment:
This method is never used by quota_usage_populating_renderer, or any other production code. It should probably be removed.

Tests for the values passed to ObjectRenderer.new, should just be moved to the test for quota_usage_populating_renderer.

Since the implementation of quota_usage_populating_renderer could change, having a test on the created_object_renderer doesn't provide much use.